### ISSUE TITLE
[Security] Review vulnerable versions of dependencies and update them to fixed versions

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#6729]
#### Description

There are a number of vulnerabilities (15 in the civiform/civiform image, 8 in the civiform/pgadmin image) due to outdated versions of dependencies referenced in the server build.sbt file, including some critical and high. You can view details by using a tool like aquasec’s trivy. We should

- Review all vulnerable versions of dependencies and update them to fixed versions
- Incorporate container scanning and alerting into build/release process

**Comment by gwendolyngoetz** [comment]
[trivvy-results.zip](https://github.com/civiform/civiform/files/14581820/trivvy-results.zip)

**Comment by gwendolyngoetz** [comment]
FYI. Tried opening the files and getting 404s. Tried upload new versions, same thing.

**Comment by gwendolyngoetz** [comment]
[This ](https://github.com/civiform/civiform/pull/6885) fixes the owasp vulnerability which is not dependent on Play 2.9+

**Comment by dkatzz** [comment]
> FYI. Tried opening the files and getting 404s. Tried upload new versions, same thing.

Weird! Do these work?

[pgadmin-latest](https://gist.github.com/dkatzz/7222f0fc366e33960589cf34e793c032)
[civiform-latest](https://gist.github.com/dkatzz/6b7be38f9f5232bb59e34bc391cd4556)

**Comment by gwendolyngoetz** [comment]
Yep. The files seems to download now too. Must have been some github nonsense

**Comment by rockycodes** [comment]
closing because it's over a year old and has not been an issue
